models/swin_unetr_btcv_segmentation/docs/README.md

```diff
@@ -27,16 +32,22 @@ Actual Model Input: 96 x 96 x 96
 ## Input and output formats
 Input: 1 channel CT image
 
-Output: 14 channels: 0:Background, 1:Spleen, 2:Right Kidney, 3:Left Kideny, 4:Gallbladder, 5:Esophagus, 6:Liver, 7:Stomach, 8:Aorta, 9:IVC, 10:Portal and Splenic Veins, 11:Pancreas, 12:Right adrenal gland, 13:Left adrenal gland
+Output: 14 channels: 0:Background, 1:Spleen, 2:Right Kidney, 3:Left Kidney, 4:Gallbladder, 5:Esophagus, 6:Liver, 7:Stomach, 8:Aorta, 9:IVC, 10:Portal and Splenic Veins, 11:Pancreas, 12:Right adrenal gland, 13:Left adrenal gland
 
 ## Performance
+
+The figure shows the training loss curve for 10K iterations.
+
+
+<p align = "center"><img src="./imgs/monai_swin_unetr_btcv_segmentation_trainloss_v1.png" alt="drawing" width="700"/></p>
+
 A graph showing the validation mean Dice for 5000 epochs.
 
-![](https://developer.download.nvidia.com/assets/Clara/Images/monai_swin_unetr_segmentation_val.png) <br>
+<p align = "center"><img src="./imgs/monai_swin_unetr_btcv_segmentation_validation_meandice_v1.png" alt="drawing" width="700"/></p>
```

Comment:
Thanks for the updates. Could you upload the images first, and then attach the url instead? Thanks!